docs/source/app-dev/services.rst

```diff
@@ -0,0 +1,159 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+The Ledger API services
+#######################
+
+
+The Ledger API is structured as a set of services. The core services are implemented using `gRPC <https://grpc.io/>`__ and `Protobuf <https://developers.google.com/protocol-buffers/>`__, but most applications will not use this low-level API, instead using the language bindings (e.g. `Java <#java>`__, `Javascript <#javascript>`__).
+
+This page gives more detail about each of the services in the API, and will be relevant whichever way you're accessing it.
```

Comment:
Why is it relevant if I won't be using the API, as you suggest in the above paragraph?